tests/test_saltworker.py

```diff
@@ -355,6 +355,49 @@ def test_linux_salt_content_none(
     assert mock_yload.call_count == 1
     assert mock_copytree.call_count > 1
 
+@pytest.mark.skipif(sys.version_info < (3, 4),
+                    reason="Not supported in this Python version.")
+@patch("codecs.open", autospec=True)
+@patch("os.walk", autospec=True)
+@patch("yaml.safe_dump", autospec=True)
+@patch("yaml.safe_load", autospec=True)
+@patch("watchmaker.utils.copytree", autospec=True)
+@patch("os.listdir", autospec=True)
+@patch("shutil.move", autospec=True)
+def test_linux_salt_content_path_none(
+        mock_move, mock_listdir, mock_copytree, mock_yload,
+        mock_ydump, mock_os, mock_codec):
+    """Test that Pythonic None can be used without error rather than 'None'."""
+    # setup ========================
+    system_params = {}
+    salt_config = {}
+    system_params["prepdir"] = "96003f32-5808-4ef8-a573-763b7f47ba9d"
+    system_params["logdir"] = "0585f9d7-ed0e-4a1b-ac0d-2b10a245a0eb"
+    system_params["workingdir"] = "35f411db-355b-4953-ae31-d6f592753e58"
+
+    salt_config["salt_content"] = "d002be6e-645d-4f58-97c9-8335df0ff5e4.zip"
+    salt_config["salt_content_path"] = "05628e08-f1be-474d-8c12-5bb6517fc5f9"
```

Comment:
The test name and description track, `test_linux_salt_content_path_none`, but we're not setting `salt_content_path` to `None` here? Maybe two tests are needed?

Reply:
yeah, I had started to test the arg to `None` and then evolved to adding more test coverage. 2 separate tests make sense.